### PR TITLE
Adapter changes for debugging go-amqp library

### LINKED ISSUE #2657: Changes for debugging adapter's go-amqp library
### Describe your problem(s)
There are sudden failures related to go-apqp, which is failing to receive messages from ASB.

### Describe your solution
Adding `debug tags` for go build and set `DEBUG_LEVEL=0` in dockerfile.

### PR DESCRIPTION
**Pull request by VajiraPrabuddhaka**
### Purpose
Adds some debug tags and log fixes to debug go-amqp.

### Issues
Fixes #2657 

### Automation tests
 - Unit tests added: N/A
 - Integration tests added: N/A

### Tested environments
Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/src/main/resources/Dockerfile [hunk 1]
```diff
@@ -56,4 +56,7 @@ COPY maven/adapter-linux-amd64 adapter
 COPY maven/check_health.sh .
 COPY maven/LICENSE.txt .
 
+# DEBUG_LEVEL is for debugging go-amqp library.
+ENV DEBUG_LEVEL=0
```
Comment (VirajSalaka) [inline adapter/src/main/resources/Dockerfile:60]: Shall we add the same to the ubuntu image as well?

Comment (VajiraPrabuddhaka) [reply adapter/src/main/resources/Dockerfile:60]: Fixed in https://github.com/wso2/product-microgateway/pull/2658/commits/a82fced9ee0645ba7522cacf42b3b365b5e89122

**Review by pubudu538 — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.11

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[failed] dev-deployment-v2 : 20220210.12

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220210.20

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220210.4

**Pull request merged**